Swope new

swopebe opened the pull request.

swopebe committed:
stolostron/backlog#20291--editing and adding

swopebe committed:
stolostron/backlog#20291--editing and adding

cadawson committed:
stolostron/backlog#20291 edits

cadawson reviewed (approved):
Thanks Brandi and Mikela, but especially Brandi on this one! It looks great! I made a few updates, so I'm good.

cadawson committed:
stolostron/backlog#20291 minor hyper update

dockerymick commented on release_notes/whats_new.adoc:
```diff
@@ -88,20 +86,32 @@ See the following cluster features that are available as Technology Preview:
-* The `hypershift` add-on helps you host OpenShift control planes at scale, separating management from workloads. The component is disabled by default.
+* Use the `hypershift` add-on to manage and provision clusters with hosted control planes at scale, separating management from workloads. The component is disabled by default.
 
   ** See the _multicluster engine operator_ documentation at link:../multicluster_engine/addon_managed_service.adoc#hypershift-addon-intro[Hypershift add-on (Technology Preview)] to learn more.
```
I don't see this file in the clusters folder: `hypershift_intro.adoc`

dockerymick commented on release_notes/whats_new.adoc:
```diff
@@ -88,20 +86,32 @@ See the following cluster features that are available as Technology Preview:
-* The `hypershift` add-on helps you host OpenShift control planes at scale, separating management from workloads. The component is disabled by default.
+* Use the `hypershift` add-on to manage and provision clusters with hosted control planes at scale, separating management from workloads. The component is disabled by default.
 
   ** See the _multicluster engine operator_ documentation at link:../multicluster_engine/addon_managed_service.adoc#hypershift-addon-intro[Hypershift add-on (Technology Preview)] to learn more.
```
I don't see this file in the clusters folder: `hypershift_intro.adoc`

swopebe replied:
@dockerymick this is a section in a file in MCE, that's why. The folder is MCE. There should be some hypershift doc but we will prob have to deal with that when it comes up if it doesn't exist.

swopebe replied:
https://github.com/stolostron/rhacm-docs/blob/2.5_stage/multicluster_engine/adv_config_install.adoc#hypershift-add-on-technology-preview

dockerymick replied:
@swopebe thanks! I think we should reference the user to the link from your previous comment, rather than pointing users to the addon_managed_service.adoc file

swopebe replied:
Not sure what you are saying.

dockerymick replied:
Reference users to `multicluster_engine/adv_config_install.adoc` instead of `multicluster_engine/addon_managed_service.adoc`

swopebe replied:
Oh, I see what you are saying...sorry, way too many links. I think it is correct now.

dockerymick committed:
Update whats_new.adoc

fixing some links. Not sure why the link to Subscription reports returns a 404.

swopebe committed:
Update whats_new.adoc

swopebe committed:
Update whats_new.adoc

swopebe committed:
Merge branch '2.5_stage' into swope-new
